bitcoin.md

```diff
@@ -52,13 +52,13 @@ $ sudo make install
 ```
 $ sudo bash -c 'cat <<EOF> /etc/bitcoin/bitcoin.conf
 server=1
+daemon=1
 disablewallet=0
-datadir=/bitcoin
-rpcuser=btcuser
-rpcpassword=btcpass
-rpcallowip=0.0.0.0/0
-bind=0.0.0.0:8333
-rpcbind=0.0.0.0:8332
+rpcuser=1
+rpcpassword=1
```

Comment:
revert

Reply:
> revert

why thats pretty simple....

Reply:
its protect by rpcallowip=127.0.0.1 doesnt matter what the default user and password is


really the old instructions are really really dangeous to use 0.0.0.0/0 for allow rpcallowip because unsuspecting users are just opening their rpc wallet commands enitrely to any connection on the internet especially if they arent behind a router or a firewall they could get attacked and someone could easily use the default username and password to do a remote rpc sendtoaddress command and send every single thing out of a user's walllet removely if the user is unsuspecting anything


really rpcallowip=127.0.0.1 keeps the rpc commands safe inside of the localhost where it isnt exposed to the WAN at all

Reply:
https://github.com/bitnet-io/stacks-core-bitnet/releases/tag/Stacks-2.0-mining-for-Bitcoin-people